frontend/test/pytest/test_jax_linalg.py

```diff
@@ -0,0 +1,1068 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
```

Comment:
you can disable pylint manually. See https://github.com/PennyLaneAI/catalyst/blob/main/frontend/test/pytest/test_adjoint.py#L30

Reply:
Thanks! Fixed.